Vizzuality/green-energy-data-platform
# Feature/language select styles

Footer (first approach) 
- Pages not created for links yet
- Missing Logo, Privacy policy and Terms and conditions

Language select styled

## Testing instructions

Change language in Language Selector (There are translations just for "Select language string si it should be the only one changing")

## Jira

https://vizzuality.atlassian.net/browse/GECHINA-33?atlOrigin=eyJpIjoiNjU5NDEzNmM2ZDQ2NDgwZDlkN2ViZmZlZTU1NWIzMzMiLCJwIjoiaiJ9
